PeerioTechnologies/peerio-icebear
# fix: optimize chat list loading [ch17654]

#### Relevant info and issue/PR links 

https://app.clubhouse.io/peerio/story/17654/fix-one-by-one-loading-and-redrawing-of-chat-list
 
#### Testing instructions  

1. Make sure chats load.
2. On Android, check if initial chat list loading is not loading rooms one-by-one as it was before.

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
